shared/js/font_size_utils.js

```diff
@@ -208,12 +216,22 @@
       var resultWidth;
       var overflowCount = -1;
 
+      if (string.length > this._overflowCountCache) {
```

Comment:
Let's add a comment here like above explaining what we are doing with the cache, just like you did above with the `_overflowCountCache`.